src/test/java/com/forest/mytopmovies/service/user/impl/TokenServiceImplUnitTest.java

```diff
@@ -0,0 +1,77 @@
+package com.forest.mytopmovies.service.user.impl;
+
+import com.forest.mytopmovies.config.ClockConfiguration;
+import com.forest.mytopmovies.constants.JwtConstants;
+import com.forest.mytopmovies.service.user.TokenService;
+import com.forest.utils.UnitTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TokenServiceImplUnitTest extends UnitTest {
+
+    private TokenService underTest;
```

Comment:
Avoid generic names like underTest because it makes it harder to keep track of what's being tested without double checking class names. Maintainers should get all necessary information from variable names without looking elsewhere to provide context.